UPGRADE-GUIDE.md

```diff
@@ -0,0 +1,1298 @@
+- `--pf-c-button--m-secondary--BorderColor` to `--pf-c-button--m-secondary--after--BorderColor`
+- `--pf-c-button--m-secondary--hover--BorderColor` to `--pf-c-button--m-secondary--hover--after--BorderColor`
+- `--pf-c-button--m-secondary--focus--BorderColor` to `--pf-c-button--m-secondary--focus--after--BorderColor`
+- `--pf-c-button--m-secondary--active--BorderColor` to `--pf-c-button--m-secondary--active--after--BorderColor`
+- `--pf-c-button--m-tertiary--BorderColor` to `--pf-c-button--m-tertiary--after--BorderColor`
+- `--pf-c-button--m-tertiary--hover--BorderColor` to `--pf-c-button--m-tertiary--hover--after--BorderColor`
+- `--pf-c-button--m-tertiary--focus--BorderColor` to `--pf-c-button--m-tertiary--focus--after--BorderColor`
+- `--pf-c-button--m-tertiary--active--BorderColor` to `--pf-c-button--m-tertiary--active--after--BorderColor`
+
+Removed variables:
+- `--pf-c-button__icon--MarginRight`
+- `--pf-c-button__text--icon--MarginLeft`
+- `--pf-c-button--m-control--disabled--after--BorderTopColor`
+- `--pf-c-button--m-control--disabled--after--BorderRightColor`
+- `--pf-c-button--m-control--disabled--after--BorderBottomColor`
+- `--pf-c-button--m-control--disabled--after--BorderLeftColor`
+
+### Card
+- Renamed vars, updated docs [(#2759)](https://github.com/patternfly/patternfly/pull/2759)
+- Moved `__header` to `__title`, `__head` to `__header` [(#3035)](https://github.com/patternfly/patternfly/pull/3035)
+
+Updated markup:
+- `.pf-c-card__header` has been renamed to `.pf-c-card__title`.
+- `.pf-c-title` and the title size modifiers should be removed from `.pf-c-card__title`.
+- `.pf-c-card__head` has been renamed to `.pf-c-card__header`.
+- `.pf-c-card__head-main` has been renamed to `.pf-c-card__header-main`.
+
+Renamed variables:
+- `--pf-c-card--m-compact__header--not-last-child--PaddingBottom` to `--pf-c-card--m-compact__header--not--last-child--PaddingBottom`
+- `--pf-c-card__header--not-last-child--PaddingBottom` to `--pf-c-card__header--not--last-child--PaddingBottom`
+- `--pf-c-card__header--not-last-child--PaddingBottom` to `--pf-c-card__header--not--last-child--PaddingBottom`
+- `--pf-c-card--m-compact__header--not--last-child--PaddingBottom` to `--pf-c-card--m-compact__title--not--last-child--PaddingBottom`
+- `--pf-c-card__header--FontFamily` to `--pf-c-card__title--FontFamily`
+- `--pf-c-card__header--FontWeight` to `--pf-c-card__title--FontWeight`
+- `--pf-c-card__header--not--last-child--PaddingBottom` to `--pf-c-card__title--not--last-child--PaddingBottom`
+- `--pf-c-card__header--not--last-child--PaddingBottom` to `--pf-c-card__title--not--last-child--PaddingBottom`
+
+### Chip
+- Refactor styles [(#2941)](https://github.com/patternfly/patternfly/pull/2941).
+- Removed hover, active, focus, and disabled from workspace [(#2975)](https://github.com/patternfly/patternfly/pull/2975).
+- Adjusted chip top/bottom padding, chip group margin in select [(#3079)](https://github.com/patternfly/patternfly/pull/3079).
```

Comment:
```suggestion
- Adjusted chip top/bottom padding and the chip group margin in select [(#3079)](https://github.com/patternfly/patternfly/pull/3079).
``` 
Did we adjust the chip top/bottom padding and the chip group margin in select? If so, edits applied are meant to add that clarity.

Reply:
yep!